src/coin/skycoin/models/wallet.go

```diff
@@ -1057,7 +1094,7 @@ func (wlt *LocalWallet) signSkycoinTxn(txn core.Transaction, pwd core.PasswordRe
 			logWallet.Errorf("Error parsing fee of TxID %s : %s", cTxn.TxID, cTxn.Fee)
 			return nil, err
 		}
-		txnFee = uint64(tmpInt64)
+		//txnFee = uint64(tmpInt64)
```

Comment:
@stdevMauricio1802 why is this change needed ?

Reply:
because when the incoming txn is a readableTxn the fee is not needed for create the returned txn